### ISSUE TITLE
Debian package repository broken

### ISSUE THREAD

**Issue by andyrooster** [fluent/fluent-bit#2160]
Since 4th May the I've been unable to use the fluentbit Debian repository.

An apt-get install command triggers a 404 error when trying to download https://packages.fluentbit.io/debian/dists/jessie/main/binary-amd64/Packages

However I can reach and install the repo key fine https://packages.fluentbit.io/fluentbit.key

Is the Debian repository working as expected?

The last time we used the Ansible script that does the apt-get install was on 6th Feb 2020 and it worked then. The code has not changed since and the install now fails with the above error.

I have also opened https://github.com/fluent/fluent-bit-packaging/issues/12 and commented on this old issue https://github.com/fluent/fluent-bit/issues/1314

For a period of time the whole of fluentbit.io was down and was apparently due to a DoS attack. This was fixed and the server started responding again and loading the fluentbit.io site but the Debian repository still doesn't work. I expect the same goes for the Ubuntu package repository.

**Comment by andyrooster** [comment]
Hello, I'm pretty certain its not my setup that is the problem as I have tried on multiple different machines and from different locations.

Could someone on the fluent bit team confirm this is a problem and let me know if this is intended to be fixed any time soon as this is blocking us from building containers.

**Comment by edsiper** [comment]
is your apt sources file pointing to this ?

```
deb https://packages.fluentbit.io/debian/jessie jessie main
```

ref: https://docs.fluentbit.io/manual/installation/linux/debian#debian-8-jessie

**Comment by andyrooster** [comment]
Ok I have found the issue. You are correct the URL I was using was a previous one. The installation documentation changed between v0.12 and v0.13, in v0.12 the APT source was listed as `deb https://packages.fluentbit.io/debian jessie main` and from v0.13 onwards it is `deb https://packages.fluentbit.io/debian/jessie jessie main` as you mention.

I suspect that there was some web server rewrite rules that were implemented when the repositories were moved and that sometime between 6th Feb 2020 (our last successful build using the old URL) and now that these rewrite rules were removed which caused our builds to start failing.

I can confirm that using the new repo URL works as expected.
